lib/arelastic/nodes/node.rb

```diff
@@ -13,6 +13,10 @@ def convert_to_elastic(expr)
         end
       end
 
+      def stringify_options(options)
+        options.each_with_object({}) { |(k, v), memo| memo[k.to_s] = v }
+      end
```

Comment:
I am skeptical of this addition. Are you trying to make all calls to `as_elastic` return string keys? If so, then this must be applied to all nodes and not the ones in this PR.

Reply:
This is a result of using kwags, and trying to use an `**options` which only allows for symbolized keys. If I remove the use of kwargs it shouldn't be necessary.